Detectors/MUON/MID/Raw/include/MIDRaw/CrateMapper.h

```diff
@@ -0,0 +1,51 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MIDRaw/CrateMapper.h
+/// \brief  Mapper to convert the RO Ids to a format suitable for O2
+/// \author Diego Stocco <Diego.Stocco at cern.ch>
+/// \date   15 November 2019
+#ifndef O2_MID_CRATEMAPPER_H
+#define O2_MID_CRATEMAPPER_H
+
+#include <cstdint>
+#include <map>
+
+namespace o2
+{
+namespace mid
+{
+class CrateMapper
+{
+ public:
+  CrateMapper();
+  ~CrateMapper() = default;
+
+  /// Builds the local board ID in the detection element
+  static uint16_t deBoardId(uint8_t rpcLineId, uint8_t columnId, uint8_t lineId) { return (rpcLineId | (columnId << 4) | (lineId << 7)); }
```

Comment:
I know we already discussed this at some point and that maybe not everyone agrees with that, but I would use free functions instead of static member functions for functions that do not use data members at all.

Reply:
But these functions are used exclusively with the mapper. So it makes not much sense to move them as free function. If I do, I should make sure to provide a unique name that does nothing conflict with any other free function. The easiest way would be to group them in a namespace...but I do not see much the difference between enclosing them in a class (that handles its input/output) w.r.t. incapsulating them in a namespace that I need to call together with CrateMapper in any case...

Reply:
Agree. Having this function inside a class tells us immediately to which service it belongs. Don't see why free functions should be preferred.

Reply:
we'll leave it as is for the moment but my point was that those 3 methods have in fact nothing to do with the crate. Might even be more an implementation detail that something that should end up in the interface, but as it's minor, let's forget about this and move on.